com.minres.coredsl/src/com/minres/coredsl/util/TypedBigInteger.java

```diff
@@ -34,9 +34,9 @@ public int getSize() {
 	public boolean isSigned() {
 		return signed;
 	}
-
-	// TODO these implementations don't make much sense to me
 	
+	// TODO these implementations don't make much sense to me
+	/*
```

Comment:
I stumbled across this in Treenail as well, so I'm in favor of dropping it.

@eyck: I presume your generators rely on this behavior somehow; would it be a great hassle for you to move this or similar logic to your code base?

Reply:
The reason I had to remove them is that it was previously impossible to obtain the original, non-tampered-with string representation of a `TypedBigInteger` and the test framework needs that to compare the actual value to the expected one.
I don't mind keeping the logic as separate functions you can call in the backend, but overwriting the default `toString` is a no-go in my eyes. I also believe it would be more appropriate to move the logic to a separate `FormattingHelper` class in the backend, since we have no use for it in the frontend.